flexx/webruntime/qtwebkit.py

```diff
@@ -35,8 +39,13 @@
 size = {size}
 pos = {pos}
 
-app = QtGui.QApplication([])
-m = QtWebKit.QWebView(None)
+if not qt == 'pyqt5':
```

Comment:
Can you change this to `if qt != 'pyqt5'`, or `if qt == 'pyqt5' and handle pyqt4/pyside in the `else` clause?